ui-tests/src/test/java/org/openmrs/reference/RecordMergeIssueTest.java

```diff
@@ -76,7 +78,7 @@ public void recordMergeIssueTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
-        headerPage.clickOnHomeIcon();
+    	homePage.go();
```

Comment:
Worth looking into the indention

Reply:
Identation is the same here try to view the file using github view file